src/App.js

```diff
@@ -3,25 +3,29 @@ import api from "./api/api";
 import { AnimatePresence } from "framer-motion";
 import { Route, Routes, useLocation, useNavigate } from "react-router-dom";
 import { default as EventEmitter } from "./event/eventEmitter";
+import { useDispatch } from "react-redux";
 
 import PageLoader from "./components/PageLoader";
 import SignUp from "./components/screens/SignUp";
 
 import "./styles/GlobalParam.css";
 import "./styles/themes/DarkTheme.css";
 import "./styles/themes/DefaultTheme.css";
+import { updateState } from "./store/ConnectState";
```

Comment:
let's rename it to `updateNetworkState`, so it  will be clearer which state we update